Week3/aalhommada.js

```diff
@@ -0,0 +1,239 @@
+//---------1. Strings!---------
+
+let myString = 'hello,this,is,a,difficult,to,read,sentence';
+console.log(myString);
+console.log(myString.length);
+let newString = myString.split(',').join(' ');
+// another way to do the previous step :
+// myString = myString.replace(/,/g , " ");
+console.log(newString);
+console.log(newString.length);
+
+//------------2. Arrays!-----------
+
+(' use strict');
+const favoritAnimals = ['blowfish', 'capricorn', 'giraffe'];
+console.log(favoritAnimals);
+favoritAnimals.push('turtle');
+console.log(
+  `The new array after adding Mauro's favorite animal 'turtle' will be ${favoritAnimals}`,
+);
+console.log(favoritAnimals);
+favoritAnimals.splice(1, 0, 'meerkat');
+console.log(
+  `The new array after adding Jim's favorite animal to the array ,it will be placed after 'blowfish' and before 'capricorn'. : ${favoritAnimals}`,
+);
+console.log(favoritAnimals);
+console.log('The array has a length of: ' + favoritAnimals.length);
+// or another way to do the previous step :
+//console.log(`the length is : ${favoritAnimals.length}`);
+favoritAnimals.splice(3, 1);
+// or :
+//favoritAnimals.splice(favoritAnimals.indexOf("giraffe"),1)
+console.log(favoritAnimals);
+let foundMeerkat = favoritAnimals.indexOf('meerkat');
+console.log('The item you are looking for is at index: ' + foundMeerkat);
+
+//---------JavaScript 🎉-----------
+//------------1----------------
+
+(' use strict');
+function arguments(arg1, arg2, arg3) {
```

Comment:
good, it's more clear what your function will do if you give it a clearer name. 
For example: `function sum(arg1, arg2,arg3) {`